clients/client-go/creds_test.go

```diff
@@ -122,6 +124,114 @@ func Test_NamedTempCred(t *testing.T) {
 		"jimmy", []string{"scope:1", "scope:2"})
 }
 
+func Test_PermaCred_Bewit(t *testing.T) {
+	rootURL := os.Getenv("TASKCLUSTER_ROOT_URL")
+	if rootURL == "" {
+		t.Skip("Cannot run test, TASKCLUSTER_ROOT_URL is not set to a non-empty string")
+	}
```

Comment:
I think it would be better to require an opt-out environment variable to indicate that the test shouldn't run, rather than an opt-in. The danger of opt-in is that a user doesn't know that `TASKCLUSTER_ROOT_URL` needs to be set, or worse, in our CI, for some reason, `TASKCLUSTER_ROOT_URL` doesn't get set when the task runs due to an accidental bug, and we don't notice that certain unit tests no longer run. On the other hand, if we require env var `SKIP_INTEGRATION_TESTS` to be set in order to skip this test and others that require a taskcluster deployment to talk to, then we can be sure that the CI will never skip tests we didn't intend it to, and users know that they are not running all tests because they had to explicitly enable the option.

Reply:
I just copied this from the existing tests -- we could change that in a followup, if you'd like to file an issue or bug.

Reply:
(the monorepo uses `NO_TEST_SKIP` as an indicator that skipped tests should fail)